wrappers/services.go

```diff
@@ -138,6 +138,13 @@ TasksAccounting=true
 	return buf.String()
 }
 
+func formatLogNamespaceSlice(grp *quota.Group) string {
+	if grp.JournalLimit == nil {
+		return ""
+	}
+	return fmt.Sprintf("LogNamespace=snap-%s\n", grp.Name)
```

Comment:
Sorry if I'm asking this a bit late. How is the feature going to work with `snap logs` or the regular debug operations with the used of a journal? AFAIU the log output is no longer part of the main journal, but needs to be accessed using `journalctl --namespace foo`.

Reply:
Really good question - the support for this will be added in another PR, I will modify the logs command to support this by supplying the namespace if the snap is a part of one